galaxy/xcms_retcor/abims_xcms_retcor.xml

```diff
@@ -319,6 +319,10 @@ Output files
 Changelog/News
 --------------
 
+**Version 2.2.0-1.50.1 - 23/03/2017**
+
```

Comment:
Macro are sometime rather hard to follow.
And that way, it's easier for me to copy/paste in the README.md shiped with the tool for the TS and GitHub